### DISCUSSION TITLE
请问一下，为何在chrome中似乎无法将其添加入脚本？问题如下

### DISCUSSION THREAD

**Discussion by GehirnZ** [the1812/Bilibili-Evolved#1645] [提问 / Q&A] [answered]
直接点下载按钮时会转到下载界面然后试图给你添加到应用程序，接着显示无法添加。
在油猴内部通过添加地址进行添加显示无法分析。
用的红猴
希望有解答，谢谢！

**Comment by the1812** [comment, marked as answer]
https://cdn.jsdelivr.net/gh/the1812/Bilibili-Evolved@master/bilibili-evolved.user.js

![image](https://user-images.githubusercontent.com/26504152/111056853-77e34300-84bd-11eb-8410-94efbc609281.png)

**Reply by GehirnZ** [reply]
这个试过了，几个版本都尝试一遍，油猴显示无法解析，还是不能安装。

**Reply by the1812** [reply]
无法解析是什么样子, 截个图看看?

**Reply by GehirnZ** [reply]
![$F`UIBM2T))4NI@%~CW IR2](https://user-images.githubusercontent.com/75194695/111057346-d4e0f800-84c1-11eb-8aae-daf8714185ad.png)
![WLZ97O13HXS{H%XJHRWF8L9](https://user-images.githubusercontent.com/75194695/111057351-dc080600-84c1-11eb-9b27-e0e3361e665d.png)
添加URL之后就是无法点安装按钮，之后就是无法解析了

**Reply by the1812** [reply]
那试试把文件下载到本地 然后油猴里新建一个脚本 把文件里的内容粘贴进去

**Reply by GehirnZ** [reply]
我记事本里打开了下的jave文件，然后黏贴扔进去试了一下还是不行，虽然这里能显示有了脚本，但是没有实际效应，和黏贴脚本之前没有变化。
![image](https://user-images.githubusercontent.com/75194695/111057926-f217c580-84c5-11eb-83a4-552788e9ec92.png)

**Reply by the1812** [reply]
有了脚本的情况下, 去b站看看控制台输出(Ctrl + Shift + I)

**Reply by GehirnZ** [reply]
调了一下好像可以了，谢谢，解决了
